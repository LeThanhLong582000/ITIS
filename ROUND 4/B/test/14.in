5465432146541