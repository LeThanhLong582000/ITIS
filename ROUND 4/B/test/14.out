-2732716073271